rustworkx/rustworkx.pyi

```diff
@@ -42,6 +39,11 @@ if sys.version_info >= (3, 13):
 else:
     from typing_extensions import TypeVar
 
+if sys.version_info >= (3, 11):
```

Comment:
This is not strictly necessary but I noticed we could refactor this to avoid depending on `typing_extension` for 3.13